Mathlib/RingTheory/Valuation/ValuationSubring.lean

```diff
@@ -35,6 +35,15 @@ structure ValuationSubring extends Subring K where
   mem_or_inv_mem' : ∀ x : K, x ∈ carrier ∨ x⁻¹ ∈ carrier
 #align valuation_subring ValuationSubring
 
+/-- If `K` is an `R`-algebra, `Place K R` is the collection of valuation subrings in `K`
+that are `R`-subalgebras. It can be given a locally ringed space structure,
+in which setting it is known as the Zariski--Riemann space. -/
+structure Place (R) [CommSemiring R] [Algebra R K]
+  extends ValuationSubring K, Subalgebra R K
```

Comment:
Hi @adamtopaz Do you think (1) this name is good, (2) the argument order is okay (or do you prefer `Place R K`)? This file actually imports TopologicalSpace so we could define the topology on this type, but it's already almost 1000 lines, so maybe put that into another file that constructs the LocallyRingedSpace / Scheme (transc. deg. 1 case) instances.